Source/Initialization/WarpXInitData.cpp

```diff
@@ -416,9 +416,9 @@ WarpX::InitLevelData (int lev, Real /*time*/)
     }
 
     if (costs[lev]) {
-        std::fill((*costs[lev]).begin(),
-                  (*costs[lev]).end(),
-                  0.0);
+        for (int i : costs[lev]->IndexArray()) {
+            (*costs[lev])[i] = 0.0;
```

Comment:
ah, too bad this does not have an `->at()` method :)
```suggestion
            costs[lev]->at(i) = 0.0;
```